membership-attribute-service/app/models/Attributes.scala

```diff
@@ -18,13 +18,15 @@ object ContentAccess {
 }
 
 case class Attributes(
-                       UserId: String,
-                       Tier: Option[String] = None,
-                       MembershipNumber: Option[String],
-                       AdFree: Option[Boolean] = None,
-                       CardExpirationMonth: Option[Int] = None,
-                       CardExpirationYear: Option[Int] = None,
-                       ContributionFrequency: Option[String] = None) {
+  UserId: String,
+  Tier: Option[String] = None,
+  MembershipNumber: Option[String],
+  AdFree: Option[Boolean] = None,
+  CardExpirationMonth: Option[Int] = None,
+  CardExpirationYear: Option[Int] = None,
+  ContributionFrequency: Option[String] = None,
+  MembershipJoinDate: Option[LocalDate] // TODO startDate shouldn't be optional once we've backfilled it everywhere
```

Comment:
MembershipJoinDate, should be optional since a person can be only a Contributor but not a Member. I would add a default value of None.